pkg/pki/k8scsrpki/k8scsr_user.go

```diff
@@ -38,12 +39,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	csrclient "k8s.io/client-go/kubernetes/typed/certificates/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 const (
 	notApprovedErrMsg         = "instance is not approved"
-	notFoundApprovedCsrErrMsg = "could not find approved csr"
+	notFoundApprovedCsrErrMsg = "could not find approved csr, or the operator is not capable of approving the csr"
```

Comment:
This is meant to be an "and", right? (according to the implementation where this error message is used)
```suggestion
	notFoundApprovedCsrErrMsg = "could not find approved csr and the operator is not capable of approving the csr"
```

Reply:
I apologize, I wrote this mechanically. Thanks for noticing.